common_module.py

```diff
@@ -80,9 +71,10 @@ def stat_write_lines(self, fout):
             fout.write('Queue: ' + qstat_values['Queue'] + '\n')
             fout.write('...\n')
 
-    def statq_write_lines(self, fout):
-        last_line = self.l.pop()
-        for qstatq_values in self.l:
+    @staticmethod
+    def statq_write_lines(all_qstatq_values, fout):
+        last_line = all_qstatq_values.pop()
```

Comment:
Non-actionable comment: I don't like the fact that it alters an argument. I would prefer it to _not_ do so by doing something like this:

    def statq_write_lines(last_line, fout):
    ...

caller:

    statq_write_lines(all_qstatq_values.pop(), ...)